### ISSUE TITLE
Websocket Support

### ISSUE THREAD

**Issue by MattNguyen** [AntonFagerberg/rackla#13]
Does Rackla support websockets? Let's say that one of my microservices exposes a websocket API, would proxying through Rackla leave a persistent connection between the client and microservice?

Also, thank you for building this out. It looks very interesting and may be a good fit for my company's API infrastructure.

**Comment by AntonFagerberg** [comment]
There is no support in Rackla for web sockets and as far as I know there is no support for it in neither Hackney nor Plug (out of the box) which Rackla is utilising - so there's no easy way of adding it to the framework, at least with the current setup.

That said, I think it is and interesting idea worth pursuing - especially to see if it can feel natural together with the existing functionality Rackla. I will start using Phoenix Channels for another project soon and hopefully there's much to be learned there.

So to sum up, currently no support, probably not in the near future but I'm very interested in suggestions and any input regarding this!
